specification/attributes/column_naming_convention.md

```diff
@@ -20,16 +18,20 @@ Naming convention for columns appearing in billing data.
 
 ## Requirements
 
-* Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
-* Column IDs MUST NOT use abbreviations.
-* Column IDs SHOULD NOT use acronyms.
-* Column IDs MUST be alphanumeric with no special characters.
-* Columns that have an ID and a Name MUST have the Id or Name suffix in the Column ID. Display Name for a Column MAY
-  avoid the Name suffix if it is considered superfluous
+- All columns defined by FOCUS MUST follow the following rules:
+  - Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
+  - Column IDs MUST NOT use abbreviations.
+  - Column IDs SHOULD NOT use acronyms.
+  - Column IDs MUST be alphanumeric with no special characters.
```

Comment:
What does "special characters" mean?  The composition of special characters changes from one platform to the next, and many such lists include underscores.  The spec should define these or link to an appropriate list.

Reply:
This line is about FOCUS columns and already exists today. We're saying that we cannot use underscores today. The one exception that's part of this PR is a vendor prefix for custom columns. If you think we should change it, can you start a thread in Slack or maybe just a separate PR? Not pushing back. Just don't want to add unrelated changes to this PR to avoid more delays.